crates/cli/tests/integration_test.rs

```diff
@@ -84,7 +84,7 @@ fn test_readme_script() {
 fn test_promises() {
     let mut runner = Runner::new("promise.js");
 
-    let (output, _, fuel_consumed) = run(&mut runner, &[]);
+    let (output, _, _) = run(&mut runner, &[]);
```

Comment:
This line is kind of unrelated to this PR but I got a warning when running the tests with the `experimental_event_loop` feature enabled so figured I'd fix the warning while I was in here.